src/views/RegionSeasonalRiskWatch/index.tsx

```diff
@@ -0,0 +1,629 @@
+import { useMemo, useState } from 'react';
+import type { FillLayer } from 'mapbox-gl';
+import { useOutletContext, useParams } from 'react-router-dom';
+import getBbox from '@turf/bbox';
+import {
+    compareNumber,
+    isDefined,
+    isFalsyString,
+    isNotDefined,
+    listToGroupList,
+    listToMap,
+    mapToList,
+    unique,
+} from '@togglecorp/fujs';
+import Map, {
+    MapSource,
+    MapLayer,
+    MapBounds,
+} from '@togglecorp/re-map';
+import {
+    defaultMapStyle,
+    defaultMapOptions,
+    adminLabelLayerOptions,
+} from '#utils/map';
+
+import MapContainerWithDisclaimer from '#components/MapContainerWithDisclaimer';
+import Container from '#components/Container';
+import {
+    getDataWithTruthyHazardType,
+    getFiRiskDataItem,
+    // getWfRiskDataItem,
+    hasSomeDefinedValue,
+    defaultApplicableHazards,
+    getValueForSelectedMonths,
+    hazardTypeToColorMap,
+    getWfRiskDataItem,
+    riskScoreToCategory,
+} from '#utils/risk';
+import type {
+    HazardType,
+    RiskDataItem,
+    RiskMetric,
+    RiskMetricOption,
+} from '#utils/risk';
+import { useRiskRequest } from '#utils/restRequest';
+import { formatNumber, maxSafe, sumSafe } from '#utils/common';
+import {
+    COLOR_LIGHT_BLUE,
+    COLOR_LIGHT_GREY,
+    COLOR_PRIMARY_RED,
+} from '#utils/constants';
+import { type RegionOutletContext } from '#utils/outletContext';
+
+import Filters from './Filters';
+import type { FilterValue } from './Filters';
+import styles from './styles.module.css';
+
+// eslint-disable-next-line import/prefer-default-export
+export function Component() {
+    const { regionResponse } = useOutletContext<RegionOutletContext>();
+    const bbox = useMemo(
+        () => (regionResponse ? getBbox(regionResponse.bbox) : undefined),
+        [regionResponse],
+    );
```

Comment:
Why do we use getBbox over bbox to get bbox?

Reply:
The type is also being resolved as 'any'

Reply:
bbox from server is not compatible with the bbox in mapbox. We need to do `yarn install` or `yarn patch-package` for the types to be resolved properly